### PR TITLE
add country module and details

### PR DESCRIPTION
**Pull request by YiruDing**
Hi, I set the relationship as one-many, and will add the test specs later today

### CODE REVIEW

**Review by alejvillanueva — CHANGES_REQUESTED** [review]
Just some small changes I think would benefit the code

File: bin/seed.js [hunk 1]
```diff
@@ -1,20 +1,52 @@
...
         const Puff = await Product.create({
             title: 'Strawberry Puff',
             brand: 'I-Mei',
             description: 'Crispy puff shell cookie with cream filling.',
             //sweet
             price: 3.99,
             inventory: 100,
-            country: 'Taiwan',
+            location: 'Taiwan',
+            countryId: 9,
```
[the remark this answers is not in the document]
Comment (YiruDing) [reply bin/seed.js:49]: Sure will do it tonight!So I just have to reset the model,then put "countryId:Taiwan.id"???

File: server/db/models/Product.js [hunk 2]
```diff
@@ -25,7 +30,7 @@ const Product = db.define('product', {
         },
     },
     inventory: { type: INTEGER, defaultValue: 0 },
-    country: {
+    location: {
```
[the remark this answers is not in the document]
Comment (YiruDing) [reply server/db/models/Product.js:33]: Ok, so I will delete the 'location' after setting 'countryId'

File: server/db/models/Product.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
-const { STRING, TEXT, INTEGER, DECIMAL } = require('sequelize')
+const { STRING, TEXT, UUID, INTEGER, DECIMAL } = require('sequelize')
 
 const db = require('../db')
 
 const Product = db.define('product', {
+    // id: {
```
Comment (alejvillanueva) [inline server/db/models/Product.js:6]: When you can, I would make this an UUID as well

Comment (YiruDing) [reply server/db/models/Product.js:6]: Sure!

File: bin/seed.js [hunk 4]
```diff
@@ -1,23 +1,55 @@
...
+            flag: 'em-kr',
+        })
+        const Spain = await Country.create({ name: 'Spain', flag: 'em-es' })
+        const Taiwan = await Country.create({
+            name: 'Taiwan',
+            flag: 'em-flag-tw',
+        })
+        const Thailand = await Country.create({
+            name: 'Thailand',
+            flag: '<em-flag-th',
```
Comment (kfless12) [inline bin/seed.js:43]: There is an extra "<" symbol here is that done on purpose?

**Review by rfmcmillan — APPROVED** [review]
Looks good! I agree with Alejandra's edit to x out the one failing test spec for now. I'll adjust the test that's failing to expect a UUID as soon as this is merged.

File: bin/seed.js (hunk 4 above)
Comment (YiruDing) [reply bin/seed.js]: Already took it off,thank you!

**Pull request merged**